### ISSUE TITLE
Need help to test new build in different platforms

### ISSUE THREAD

**Issue by ozbillwang** [alpine-docker/git#69]
From 30th Sept 2024, Circle CI deprecated the Remote Docker images I am using in Circle CI pipeline

<img width="947" alt="image" src="https://github.com/user-attachments/assets/f8945bd3-a459-46b3-ae20-47497de236ae">

Ref: https://discuss.circleci.com/t/remote-docker-image-deprecations-and-eol-for-2024/50176

i have tested the new Multi-Arch builds with Github Actions (https://github.com/alpine-docker/git/actions/runs/11143992652) (https://github.com/alpine-docker/git/pull/68)

and successfully push the new image as `alpine/git:1cac1e6`

<img width="1753" alt="image" src="https://github.com/user-attachments/assets/7aca7093-ecbb-4001-9302-ed74d7296f86">

i need help to run a test in other platforms `linux/arm/v7,linux/arm64/v8,linux/arm/v6,linux/ppc64le,linux/s390x,linux/386`

Thanks.

cc: @barthy1

**Comment by ozbillwang** [comment]
from my test (on mac, intel chip), it looks fine to me

```
$ cat test.sh
for platform in linux/amd64 linux/arm/v7 linux/arm64/v8 linux/arm/v6 linux/ppc64le linux/s390x linux/386
do
  echo "Platfom is $platform"
  docker run --rm -ti --platform "${platform}" --entrypoint=sh alpine/git:1cac1e6 -c "uname -a"
done

$  ./test.sh
Platfom is linux/amd64
Linux ae980d768430 6.10.4-linuxkit #1 SMP PREEMPT_DYNAMIC Mon Aug 12 08:48:58 UTC 2024 x86_64 Linux
Platfom is linux/arm/v7
Linux 78d1ec811eed 6.10.4-linuxkit #1 SMP PREEMPT_DYNAMIC Mon Aug 12 08:48:58 UTC 2024 armv7l Linux
Platfom is linux/arm64/v8
Linux ba7a061107fc 6.10.4-linuxkit #1 SMP PREEMPT_DYNAMIC Mon Aug 12 08:48:58 UTC 2024 aarch64 Linux
Platfom is linux/arm/v6
Linux d01b9d794968 6.10.4-linuxkit #1 SMP PREEMPT_DYNAMIC Mon Aug 12 08:48:58 UTC 2024 armv7l Linux
Platfom is linux/ppc64le
Linux 1ec3cf089913 6.10.4-linuxkit #1 SMP PREEMPT_DYNAMIC Mon Aug 12 08:48:58 UTC 2024 ppc64le Linux
Platfom is linux/s390x
Linux d32f5a7951bc 6.10.4-linuxkit #1 SMP PREEMPT_DYNAMIC Mon Aug 12 08:48:58 UTC 2024 s390x Linux
Platfom is linux/386
Linux 4a0fb62e57d7 6.10.4-linuxkit #1 SMP PREEMPT_DYNAMIC Mon Aug 12 08:48:58 UTC 2024 x86_64 Linux
```

**Comment by ozbillwang** [comment]
new pipeline has been released via #70
